### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by kryvets-vadym**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+
+  const sourceStringArray = sourceString
+    .replace(/\s+/g, ' ')
```
Comment (Oleksii-Siviolov) [inline src/convertToObject.js:12]: This is nice. I forgit about this expression, but it is really a good case to use it.

File: src/convertToObject.js [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
```
Comment (Oleksii-Siviolov) [inline src/convertToObject.js:9]: Not the best way to name your object, see #2 in Checklist. Nobody will understand, what is inside of this object.

**Review by Oleksii-Siviolov — CHANGES_REQUESTED** [review]
Nicely done, I like your solution, but you should pay attention to the naming.

**Review by Oleksii-Siviolov — APPROVED** [review]
GJ! Keep it up!

**Review by mykola-koval — APPROVED** [review]
the logic could be simplified a little, but ok
check other students' work, make some student reviews :)

**Pull request closed without merging**